src/System.Runtime.InteropServices/tests/Configurations.props

```diff
@@ -2,8 +2,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
-      netcoreapp;
+      netcoreapp-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
@joperezr is this the right set of configs? The src\configurations.props has
```xml
    <BuildConfigurations>
      uapaot-Windows_NT;
      netcoreapp-Windows_NT;
      netcoreapp-Unix;
      uap-Windows_NT;
    </BuildConfigurations>
```

So I think this is right, but is there any value in the OS suffixes for the uap/uapaot ones?